frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -132,8 +135,10 @@ describe('Kubernetes resource CRUD operations', () => {
         // filter should have 3 chip groups: resource, label, and name
         listPage.filter.numberOfActiveFiltersShouldBe(3);
         listPage.rows.shouldExist(name);
-        cy.testA11y(`Search page for ${kind}: ${name}`);
-
+        // Search page same for all resources, so only a11y test once
```

Comment:
This isn't true since the lists are different for each resource. I'd go ahead and test each one unless there is a good reason not to.

Reply:
ok, thought it was the same for every resource.